### DISCUSSION TITLE
Custom data values (e.g. format currencies)

### DISCUSSION THREAD

**Discussion by iamgoodbytes** [rappasoft/laravel-livewire-tables#1248] [Q&A] [answered]
Hi! I'm looking for a way to adjust my column data like the `price`column. I currently load it up in the datatable but would like to format it before it gets shown in the table.

Examples are
- `title` column -> I'd like to truncate the value to a maximum of e.g. 10 characters and then add ...
- `price` -> database value is 0.00 but I'd like to show it as 0€

So in short, what would be the correct place to do these customizations?

**Comment by lrljoe** [comment]
You can use the format() option on your column.

**Reply by iamgoodbytes** [reply]
Thank you @lrljoe - that works for formatting a value. Maybe I should have asked the question in another way at once. Is it possible to for example create a combined custom column named `myCustomColumn`? 

like so:
`field1: true`
`field2: false`

I'd like to show a custom column with 0, 1 or 2 images, depending on the value of the fields. If they are set to true, I'd like to show a small icon for those columns.

**Reply by iamgoodbytes** [reply, marked as answer]
I believe `label()` is the answer https://github.com/rappasoft/laravel-livewire-tables/discussions/898
